File: test.spritebuilder/Source/TDEngine/internalObject/MapStateInternal.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package internalObject;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author ver
 */


public class MapStateInternal {

    public static List<SolderInternal> unit  = new ArrayList<SolderInternal>();
    public static List<TowerInternal> towers = new ArrayList<TowerInternal>();
    public static List<ShellInternal> shells = new ArrayList<ShellInternal>();
     
    public boolean isEnd;
    public boolean isWin;
    public int money;
 
    
  
}

File: TowerDefence.spritebuilder/internalObject/MapStateInternal.swift
//
//  MapStateInternal.swift
//  TowerDefence
//
//  Created by koveres on 02.12.14.
//  Copyright (c) 2014 Apportable. All rights reserved.
//

import Foundation


public class MapStateInternal {
    
    public   var  unit:[SolderInternal] =  [SolderInternal]();
   
    public   var  towers:[TowerInternal] =  [TowerInternal]();
    public   var  shells:[ShellInternal] =  [ShellInternal]();
    
    public   var isEnd = false;
    public   var isWin = false;
    public   var money:CInt = 0;
}
